lib/tictactoe_ui.ex

```diff
@@ -1,88 +1,175 @@
 defmodule TicTacToe.UI do
 
-  def start_game() do
-    TicTacToe.make_move()
-    |> decide_turn_type()
-  end
-  
-  def get_move(game_status) do
-    print_turn(:get_player_move, game_status) 
-  end
+  @clear_screen "\e[H\e[2J"
+
+  def start_game_loop, do: start_game |> game_loop
+
+  def start_game, do: TicTacToe.first_game_status |> do_next_turn
```

Comment:
I mentioned this in yday's IPM as well. I told you that when I was trying to run it, I intuitively thought that `start_game` was the entry point, so the naming is confusing.  Would it be more intuitive to rename `start_game` to `take_turn`, and `start_game_loop` to `start_game`?